src/Algebra/Construct/Zero.agda

```diff
@@ -13,53 +13,54 @@
 
 {-# OPTIONS --without-K --safe #-}
 
-module Algebra.Construct.Zero where
+open import Level using (Level)
+
+module Algebra.Construct.Zero {ℓ : Level} where
 
 open import Algebra.Bundles
-open import Data.Unit
-open import Level using (Level; 0ℓ)
+open import Data.Unit.Polymorphic
 
 ------------------------------------------------------------------------
 -- Raw bundles
 
-rawMagma : RawMagma 0ℓ 0ℓ
+rawMagma : RawMagma ℓ ℓ
```

Comment:
Is there something that forces these to be the same level?

Reply:
I don't think so. Thanks for the review. See previous reply.